cli/README.md

```diff
@@ -61,6 +61,17 @@ To process json file containing sequence of events :
 run `node cli.mjs --sequence "path of sequence.json"`
 example : node cli.mjs --sequence ../examples/device-onDeviceNameChanged.sequence.json
 
+## Status of ws connection
+
+To get the status of web-socket connection of user :
+run `node cli.mjs --getStatus`
```

Comment:
I think this is probably fine, but now that I think about it, it might be nice if the name of this cli param included what the status is for/about. E.g., getUserStatus? getSocketStatus?

Reply:
Changed to "--getSocketStatus" in the CLI and all documentation. I left the API alone as "/api/v1/status"